enhancements/evidence-forwarding.md

```diff
@@ -0,0 +1,320 @@
+which are bound to limitations of etcd.
+
+Unfortunately, physical volumes have their own short-comings. Reports stored in
+physical volumes still need to be fetched for evidence. Physical volumes can
+also be deleted like any other cluster resource.
+
+A improvement to the idea of putting evidence reports on disk would be to dump
+them into an object or file store. This has an added benefit of allowing
+administrators managing multiple clusters to put evidence in a single location.
+
+## Motivation
+
+The motivation of the enhancement is to store compliance evidence outside the
+cluster and its storage pools.
+
+### Goals
+
+The goal is to provide a way for Kubernetes administrators to configure the
+Compliance Operator to send compliance reports, specifically ARF reports, to an
+external storage pool, like an object or file store.
+
+### Non-Goals
+
+It is not the goal of this enhancement to implement results forwarding, or
+adjust the definition of evidence outside an ARF report.
+
+## Proposal
+
+This is where we get down to the nitty gritty of what the proposal
+actually is. Describe clearly what will be changed, including all of
+the components that need to be modified and how they will be
+different. Include the reason for each choice in the design and
+implementation that is proposed here, and expand on reasons for not
+choosing alternatives in the Alternatives section at the end of the
+document.
+
+### User Stories
+
+1. As a Kubernetes administrator, I want to configure the Compliance Operator
+   to send all ARF reports to a dedicated object store.
+2. As a Kubernetes administrator, I want to opt out of storing any ARF reports
+   on physical volumes.
+3. As a Kubernetes administrator, I expect verbose alerts when the Compliance
+   Operator cannot send ARF reports to a configured external storage pool.
+
+### API Extensions
+
+API Extensions are CRDs, admission and conversion webhooks, aggregated API servers,
+and finalizers, i.e. those mechanisms that change the OCP API surface and behaviour.
+
+- Name the API extensions this enhancement adds or modifies.
+- Does this enhancement modify the behaviour of existing resources, especially those owned
+  by other parties than the authoring team (including upstream resources), and, if yes, how?
+  Please add those other parties as reviewers to the enhancement.
+
+  Examples:
+  - Adds a finalizer to namespaces. Namespace cannot be deleted without our controller running.
+  - Restricts the label format for objects to X.
+  - Defaults field Y on object kind Z.
+
+Fill in the operational impact of these API Extensions in the "Operational Aspects
+of API Extensions" section.
+
+### Implementation Details/Notes/Constraints [optional]
+
+What are the caveats to the implementation? What are some important details that
+didn't come across above. Go in to as much detail as necessary here. This might
+be a good place to talk about core concepts and how they relate.
```

Comment:
Use this as an opportunity to investigate how we want to organize the forwarding implementation.

Note, there is probably some overlap in the interface we need here and the interface we need for forwarding results (discussed in a separate enhancement).